File: clients/fireback-android/app/src/main/java/com/fireback/modules/workspaces/EmailSenderEntity.java
package com.fireback.modules.workspaces;
import com.fireback.modules.workspaces.OkayResponseDto;
import com.google.gson.Gson;
import com.fireback.JsonSerializable;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;
public class EmailSenderEntity extends JsonSerializable {
    public String fromName;
    public String fromEmailAddress;
    public String replyTo;
    public String nickName;
    public static class VM extends ViewModel {
    // upper: FromName fromName
    private MutableLiveData< String > fromName = new MutableLiveData<>();
    public MutableLiveData< String > getFromName() {
        return fromName;
    }
    public void setFromName( String  v) {
        fromName.setValue(v);
    }
    // upper: FromEmailAddress fromEmailAddress
    private MutableLiveData< String > fromEmailAddress = new MutableLiveData<>();
    public MutableLiveData< String > getFromEmailAddress() {
        return fromEmailAddress;
    }
    public void setFromEmailAddress( String  v) {
        fromEmailAddress.setValue(v);
    }
    // upper: ReplyTo replyTo
    private MutableLiveData< String > replyTo = new MutableLiveData<>();
    public MutableLiveData< String > getReplyTo() {
        return replyTo;
    }
    public void setReplyTo( String  v) {
        replyTo.setValue(v);
    }
    // upper: NickName nickName
    private MutableLiveData< String > nickName = new MutableLiveData<>();
    public MutableLiveData< String > getNickName() {
        return nickName;
    }
    public void setNickName( String  v) {
        nickName.setValue(v);
    }
    }
}

File: modules/workspaces/codegen/ios/firebackios/modules/workspaces/EmailSenderEntity.swift
class EmailSenderEntity : Codable {
    private var _fromName: String?
    var `FromName`: String? {
        set { _fromName = newValue }
        get { return _fromName }
    }
    private var _fromEmailAddress: String?
    var `FromEmailAddress`: String? {
        set { _fromEmailAddress = newValue }
        get { return _fromEmailAddress }
    }
    private var _replyTo: String?
    var `ReplyTo`: String? {
        set { _replyTo = newValue }
        get { return _replyTo }
    }
    private var _nickName: String?
    var `NickName`: String? {
        set { _nickName = newValue }
        get { return _nickName }
    }
}